cmd/target-browser/cmd/constant.go

```diff
@@ -76,7 +76,8 @@ const (
 	expiryDateUsage   = "Backup expiry date"
 
 	OperationScopeFlag  = "operation-scope"
-	operationScopeUsage = "Filter backup/backupPlan for [SingleNamespace, MultiNamespace]"
+	operationScopeUsage = "Filter backup/backupPlan for [SingleNamespace, MultiNamespace]. " +
+		"Supported values: SingleNamespace|MultiNamespace or singleNamespace|multiNamespace"
```

Comment:
supported values can be in any case. just characters should match that's it. Change desc.